src/osd/OSD.cc

```diff
@@ -8185,6 +8224,8 @@ void OSD::ShardedOpWQ::_process(uint32_t thread_index, heartbeat_handle_d *hb )
     }
   }
   pair<PGRef, PGQueueable> item = sdata->pqueue.dequeue();
+  sdata->ql_accum += sdata->pqueue.empty() ? 1 : 2; 
```

Comment:
what do you think maintaining the ql_accum by ShardedOpWQ itself instead of getting length from pqueue.